src/streaming/controllers/BufferController.js

```diff
@@ -152,8 +153,10 @@ function BufferController(config) {
             errHandler.mediaSourceError('Error creating ' + type + ' source buffer.');
         }
 
+        quality = requiredQuality === -1 ? 0 : requiredQuality;
```

Comment:
@AkamaiDASH This needs addressing as it seems to be breaking SegmentBase streams. I'll open an issue.